front/app/containers/IdeasNewPage/WithJSONForm/index.tsx

```diff
@@ -113,10 +115,31 @@ const IdeasNewPageWithJSONForm = ({ params }: WithRouterProps) => {
     });
     const ideaId = idea.data.id;
 
-    clHistory.push({
-      pathname: `/ideas/${idea.data.attributes.slug}`,
-      search: `?new_idea_id=${ideaId}`,
-    });
+    // Check ParticipationMethodConfig for form submission action
+    if (!isNilOrError(phases)) {
+      // Check if URL contains specific phase_id
+      const queryParams = new URLSearchParams(window.location.search);
+      const phaseIdFromUrl = queryParams.get('phase_id');
+      const participationMethodFromURL = phases
+        .filter((phase) => phase.id === phaseIdFromUrl)
+        .map((phase) => phase.attributes.participation_method)[0];
```

Comment:
I expect that `phases` includes only 1 phase with the `phaseIdFromUrl`, so then the `map` is not needed, because `filter` will return an array with 1 element.

```suggestion
      const participationMethodFromURL = phases
        .filter((phase) => phase.id === phaseIdFromUrl)[0].attributes.participation_method;
```

Reply:
Ah yes you're right. I thought we needed a map if we are using filter for some reason. I'll use your suggestion :) Thanks!

Reply:
It seems we do need the map here - we get a type error otherwise. I'm assuming it's because the code can't tell that filter will result in only 1 response (it's something we know from context)?